ansible/group_vars/all

```diff
@@ -1,75 +1,75 @@
+66623861633038646266363532653738623834396238663165386365343461633764643839373433
+35353332396662373233643532613335613931323431346561316462363339383536656433633166
+34383664643134393664383865616131633930633966646165623430383466643433373337653430
+38343731613632353532346238633236323765663538616262323832653138633263373232373631
+39633638366663613466663633623233656132356565386266653630316633656130613834643061
+31353336353465386561386533393030306639383462663931393137633864646533383037373362
+63303033623163613131643538386566356362623733656262386533643839373532316137623932
+34303131626632383930303438363861666137363564656130303031633564333632316539343164
+64663465656165653634343838646264663433633865316165643961363130353233376266646133
+64636137383134646465373161616465613830313538616630396133316237646563373535626461
+31633636613130386238343231623830666461613739376166373965346464353134356565316630
+62366336653761356666623861646334623338386236376265313035386531363633353065346236
+33323035633962373037636431643839613430303763363162613965626639623730643561333564
+31346532376635333066636334393862383536643362666438646635643933363135363833373734
+61653835653563356539376539353065663665333135353161623336326530663861623765666339
+35323034306636316266663539646239336666396336363366373665313965363864633965393133
+33353265663630653139303931636136653635383933643239346231366536663031316338623732
+62626163313363323565333863366461626266653730323338623130323239323830656165376664
+36316262346433613939303931336662663039386261346230653635333364636662343738323163
+66613436626266633863373031376339633166396438383731333231363362353237303231343064
+36373130383764373764663763353534373661316639343036643366376662303263306336366630
+37643134643632363031303436613838343062616464336631653032363331323265333638666137
+34363736616534643137643366633466313136633761636532393630646661346331316135336134
+38306632316663313066643536313934396263653862393232353939666134353037323133356132
+65316236376334663134626333363232393734643531363433356634636339363333643934346532
+35613535383463356432343537623231623036613732323065373331373464366336333861393337
+63343033366135643938386162303364633235313762373266633063643138343963633739656531
+30333035386163333266313139643338306531666435633939613633313330313538333337363238
+63623364336564363533666539613331343536343364663061343964396365326564616561393132
+65303564616231376563626637633063386130663132643830636564616236633530363964303063
+61313531333836653234613339306335306338613638366565656362666134666636316161663130
+37313065363066616433313338396536656265636364376431333633396639613437356434356639
+63333838353735613263333436613434643430636266653366303630303437626635396238306466
+63663762633566616536303732643662623861636462353335383035356534323739653535373432
+65663435643738646265323138663933643930356335333162323563666532626531616164636165
+3961383331316235363738613433333130666234393237663430
```

Comment:
Hard to tell what was removed here - hopefully not the AWS details that I use elsewhere.